src/pages/getting-started/index.mdx

```diff
@@ -28,6 +28,18 @@ From a more tech perspective, the LiveChat Platform provides:
 - monetization tools and <Placeholder id="DEVELOPER_CONSOLE_URL"/>
 - the <Placeholder id="LIVECHAT_MARKETPLACE_URL"/> of apps and services
 
+# What HelpDesk is
+
+[HelpDesk](https://www.helpdesk.com/) is a ticketing system that helps you to manage both your asynchronous and synchronous communication with customers. You can imagine it as a mailbox with many features that make communication much more manageable:
+
+- [Canned responses](https://www.helpdesk.com/help/what-are-canned-responses/)
```

Comment:
Since these are all external links, we should use the `<a>` tag with "target blank", so that it opens in a new tab